modules/chip/ChipList.tsx

```diff
@@ -37,14 +39,18 @@ function ChipList<T>({
   list = [],
   displayFn = (item: T) => item,
   onDismissChip,
+  onChipClick,
   onListChange,
   wrappedRef,
   parentStylesheet,
   ...props
 }: IProps<T> & WithStylesProps) {
   const [_list, setList] = useState(list);
   const divRef = useRef<HTMLDivElement>(null);
-  const CloseableChipStyle = useRegistry<ICloseableChipProps>("CloseableChip");
+  const CloseableChipStyle = useRegistryWithStyles<ICloseableChipProps>(
```

Comment:
This is a weird/ misleading name. Maybe StyledCloseableChip is more accurate - what do you think?